Client/Frontend/Browser/Authenticator.swift

```diff
@@ -162,7 +166,39 @@ class Authenticator {
             textfield.text = credentials?.password
         }
 
+        self.sendLoginsAutofillPromptShownTelemetry()
         viewController.present(alert, animated: true) { () -> Void in }
         return deferred
     }
+
+    // MARK: Telemetry
+    private static func sendLoginsPasswordDetectedTelemetry() {
+        TelemetryWrapper.recordEvent(category: .information,
+                                     method: .view,
+                                     object: .loginsPasswordDetected)
```

Comment:
This won't be called in `TelemetryWrapper`. Method isn't the same (here's it's `view` but in the `TelemetryWrapper` it's `emailLogin`